sklearn/linear_model/_huber.py

```diff
@@ -242,6 +244,15 @@ class HuberRegressor(LinearModel, RegressorMixin, BaseEstimator):
     Linear Regression coefficients: [-1.9221...  7.0226...]
     """
 
+    _parameter_constraints = {
+        "epsilon": [Interval(Real, 1.0, None, closed="neither")],
```

Comment:
Yes `greater` is a bit ambiguous. Let's be precise

Reply:
@glemaitre this is because you are french :)
According to the internet, greater than does not include the number in general. I think it doesn't hurt to be extra precise to avoid any confusion